Tools/PIDML/pidOnnxModel.h

```diff
@@ -213,48 +216,104 @@ struct PidONNXModel {
     }
   }
 
+  template <typename... P1, typename... P2>
+  static constexpr bool is_equal_size(o2::framework::pack<P1...>, o2::framework::pack<P2...>)
```

Comment:
do you really need this function? Isn't it just `pack_size({some pack}) == pack_size({some othe pack})`